### PR TITLE
FIX: Parameter in example, method to create parameter for longitude a…

### PR DESCRIPTION
**Pull request by ghost**
Update parameter in jupyter notebook
Update method to create filter for latitude, longitude and bbox

### CODE REVIEW

File: fairease/udal/brokers/beacon.py [hunk 1]
```diff
@@ -87,9 +87,13 @@ def _execute_argo(self, params: dict):
 
         # Latitude and longitude
         if 'latitude' in params:
-            json_params['filters'].append({"for_query_parameter": "Latitude", "min": params['latitude'], "max": params['latitude']})
+            min = params['latitude'] - 0.5
```
Comment (jorgecunhamendes) [inline fairease/udal/brokers/beacon.py:92]: Why `- 0.5` and `+ 0.5` ? Please add a comment above (after line 88, before the if statement)

File: fairease/udal/brokers/iddas.py [hunk 2]
```diff
@@ -90,18 +90,33 @@ def _build_sparql_filter(self, params: Dict[str, Union[str, float, int]]) -> str
         if 'endTime' in params:
             sparql_filter.append(f"FILTER(BOUND(?endDate) && ?endDate <= '{params['endTime']}'^^xsd:date) .")
         if 'latitude' in params and 'longitude' in params:
-            sparql_filter.append(f"FILTER(BOUND(?bbox) && ?bbox = 'POINT ({params['latitude']} {params['longitude']})'^^geo:wktLiteral) .")
+            min_latitude = params['latitude'] - 10
+            max_latitude = params['latitude'] + 10
+            min_longitude = params['longitude'] - 10
+            max_longitude = params['longitude'] + 10
```
Comment (jorgecunhamendes) [inline fairease/udal/brokers/iddas.py:98]: Why `- 10` and `+ 10` ? Please add a comment.

**Review by jorgecunhamendes — CHANGES_REQUESTED** [review]

File: fairease/udal/brokers/iddas.py (hunk 2 above)
Comment (ghost) [reply fairease/udal/brokers/iddas.py:98]: Below +/- 10, IDDAS does not find any dataset. I don't know why because Beacon finds data

Comment (jorgecunhamendes) [reply fairease/udal/brokers/iddas.py:98]: 10 degrees is a large area... anyway, we'll keep it as is for now

**Pull request merged**